modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/info/item/provider/test/ObjectEntryInfoItemFieldValuesProviderTest.java

```diff
@@ -114,12 +114,8 @@ public static void tearDownClass() {
 	public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 
-		ServiceContext serviceContext =
-			ServiceContextTestUtil.getServiceContext(_group.getGroupId());
-
-		serviceContext.setRequest(_getHttpServletRequest(_group));
-
-		ServiceContextThreadLocal.pushServiceContext(serviceContext);
+		_serviceContext = ServiceContextTestUtil.getServiceContext(
+			_group.getGroupId());
```

Comment:
@vngoliveira this variable is only used by one test: `testObjectEntryInfoItemFieldValuesProvider`.

That said, I see no reason to make it global and add it here in the `setUp` method.